CHANGELOG.md

```diff
@@ -1,88 +1,87 @@
 
-When using the [footer](https://design-system.service.gov.uk/components/footer/) Nunjucks macro, you can now translate the text of the Open Government Licence (OGL) and Crown copyright statements using the `contentLicence` and `copyright` parameters.
+When using the [footer Nunjucks macro](https://design-system.service.gov.uk/components/footer/#options-default-1), you can now translate the text of the Open Government Licence (OGL) and Crown copyright statements using the `contentLicence` and `copyright` parameters.
 
-Visit The National Archives' [documentation on OGL and Crown copyright](https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/open-government-licence/copyright-notices-attribution-statements/) for information on what needs to be included in these statements.
+Visit [The National Archives' documentation on OGL and Crown copyright](https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/open-government-licence/copyright-notices-attribution-statements/) for information on what you need to include in these statements.
 
 This was added in [pull request #2702: Allow localisation of content licence and copyright notices in Footer](https://github.com/alphagov/govuk-frontend/pull/2702).
 
 #### Pass HTML directly into compatible components
 
-If using the Nunjucks macros, you can now pass HTML content directly into compatible components using [the Nunjucks `call` syntax](https://mozilla.github.io/nunjucks/templating.html#call). If HTML is provided through the call syntax, the `html` and `text` options will be ignored.
+If using the Nunjucks macros, you can now pass HTML content directly into compatible components using [the Nunjucks call syntax](https://mozilla.github.io/nunjucks/templating.html#call). If HTML is provided through the call syntax, the nunjucks macro will ignore the HTML and text options.
 
-Components that have been updated to support this syntax are:
+Components updated to support this syntax are:
 
-- Details
-- Error summary (mapped to `descriptionHtml` parameter)
-- Inset text
-- Notification banner
-- Panel
+-   Details
+-   Error summary (mapped to descriptionHtml parameter)
+-   Inset text
+-   Notification banner    
+-   Panel
 
 This was added in [pull request #2734: Update various components to be callable](https://github.com/alphagov/govuk-frontend/pull/2734).
 
 #### Use new override classes to apply static spacing
 
 You can now use static spacing override classes to apply spacing from [the static spacing scale](https://design-system.service.gov.uk/styles/spacing/#static-spacing) to elements of your design.
 
-The new classes start with: `govuk-!-static` followed by either `margin-` or `padding-`, and then a spacing unit number.
+The new classes start with: `govuk-!-static` followed by either margin- or padding-, and then a spacing unit number.
```

Comment:
Personally think this read easier with `margin-` and `padding-` in code quotes.

Should probably also include the ending dash on `govuk-!-static-`?